hack/test-cmd.sh

```diff
@@ -307,11 +307,11 @@ for version in "${kube_api_versions[@]}"; do
   # Post-condition: name is still valid-pod
   kube::test::get_object_assert 'pod valid-pod' "{{.${labels_field}.name}}" 'valid-pod'
 
-  ### --overwrite must be used to overwrite existing label
+  ### --overwrite must be used to overwrite existing label, can be applied to all resources
   # Pre-condition: name is valid-pod
   kube::test::get_object_assert 'pod valid-pod' "{{.${labels_field}.name}}" 'valid-pod'
   # Command
-  kubectl label --overwrite pods valid-pod name=valid-pod-super-sayan "${kube_flags[@]}"
+  kubectl label --overwrite pods --all name=valid-pod-super-sayan "${kube_flags[@]}"
```

Comment:
I was just copying the previous person's incorrectly spelled dbz reference.

----- Original Message -----
> >    # Pre-condition: name is valid-pod
> >    kube::test::get_object_assert 'pod valid-pod'
> >    "{{.${labels_field}.name}}" 'valid-pod'
> >    # Command
> > -  kubectl label --overwrite pods valid-pod name=valid-pod-super-sayan
> > "${kube_flags[@]}"
> > +  kubectl label --overwrite pods --all name=valid-pod-super-sayan
> > "${kube_flags[@]}"
> 
> Entirely frivolous nit. I figured you were making a dbz reference :P
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/GoogleCloudPlatform/kubernetes/pull/5976/files#r27233801